bin/fonto-benchmark-runner

```diff
@@ -18,39 +18,42 @@ require('ts-node').register({
 // const startServer = require('../lib/server.js').default;
 const runTests = require('../lib/runBenchmarks.js').default;
 
-// const parseArgv = require('./parseArgv');
-// const packageJson = require('../package.json');
+const parseArgv = require('./parseArgv');
+const packageJson = require('../package.json');
 
 (async function() {
-	// const { server, port, node } = await parseArgv(packageJson, process.argv, {
-	// 	description: 'Run performance benchmarks on Fonto packages in Node or in the browser',
-	// 	options: {
-	// 		server: {
-	// 			identifier: '[--server]',
-	// 			description: 'Run benchmarks in the browser',
-	// 			defaultValue: false
-	// 		},
-	// 		port: {
-	// 			identifier: '[--port <port>]',
-	// 			description: 'Set a port for the server',
-	// 			defaultValue: 8080
-	// 		},
-	// 		node: {
-	// 			identifier: '[--node]',
-	// 			description: 'Run tests in Node',
-	// 			defaultValue: false
-	// 		}
-	// 	}
-	// });
+	const { /*server, port, node,*/ isLogExcellable } = await parseArgv(packageJson, process.argv, {
+		description: 'Run performance benchmarks on Fonto packages in Node or in the browser',
```

Comment:
This description is currently not correct as we do not (yet) support running the benchmarks in the browser.